utils/errorPricingApi.ts

```diff
@@ -0,0 +1,17 @@
+import { PRICINGAPIERROR } from '@/modules/backend/openapi-generated'
+import { isError } from '@/utils/errors'
+
+export class PricingApiError extends Error {
+  errorName: PRICINGAPIERROR
```

Comment:
I know that this type is generated, but it looks awful. Is there any way to change it somewhere?